File: app/src/main/java/com/digicoffer/lauditor/Meetings/Models/TeamDo.java
package com.digicoffer.lauditor.Meetings.Models;

public class TeamDo {
    String id;
    String name;
    private boolean isSelected;
    boolean isChecked;
    boolean isenabled;

    public boolean isSelected() {
        return isSelected;
    }

    public void setSelected(boolean selected) {
        isSelected = selected;
    }

    public boolean isChecked() {
        return isChecked;
    }

    public void setChecked(boolean checked) {
        isChecked = checked;
    }

    public boolean isIsenabled() {
        return isenabled;
    }

    public void setIsenabled(boolean isenabled) {
        this.isenabled = isenabled;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;

        TeamDo d = (TeamDo) o;

        return id != null ? id.equals(d.id) : d.id == null;
    }

    @Override
    public int hashCode() {
        return id != null ? id.hashCode() : 0;
    }
}

File: app/src/main/java/com/digicoffer/lauditor/Meetings/Models/TeamDo.kt
package com.digicoffer.lauditor.Meetings.Models

class TeamDo {
    var id: String? = null
    var name: String? = null

    @get:JvmName("isSelected")
    @set:JvmName("setSelected")
    var isSelected: Boolean = false

    @get:JvmName("isChecked")
    @set:JvmName("setChecked")
    var isChecked: Boolean = false

    @get:JvmName("isIsenabled")
    @set:JvmName("setIsenabled")
    var isenabled: Boolean = false

    override fun equals(other: Any?): Boolean {
        if (this === other) return true
        if (other == null || javaClass != other.javaClass) return false

        val d = other as TeamDo

        return if (id != null) id == d.id else d.id == null
    }

    override fun hashCode(): Int {
        return id?.hashCode() ?: 0
    }
}
